src/server/pfs/server/server.go

```diff
@@ -18,26 +16,14 @@ const (
 	GoogleBackendEnvVar = "GOOGLE"
 )
 
-// APIServer represents an api server.
+// APIServer represents and internal api server.
 type APIServer interface {
-	pfsclient.APIServer // SJ: WOW this is bad naming
-	shard.Frontend
+	pfsclient.APIServer // SJ: also bad naming
 }
```

Comment:
We should just get rid of those comments altogether. 

FWIW - 

Those comments were way back from the cleint/server reorg during the vendoring refactor. The comment alludes to the fact that `pfsclient.APIServer` is a bit silly ... we have to namespace all the server itnerfaces under the client namespace for vendoring purposes ... but to someone just reading that code ... its not the clearest thing.